### ISSUE TITLE
[Style] Allow multiple style stereotypes for Mindmap, WBS, ...

### ISSUE THREAD

**Issue by The-Lum** [plantuml/plantuml#1527]
Hello PlantUML team,
- Could you allow multiple style stereotypes for Mindmap, WBS, Activity diagram, ...

## ✔️ OK on Class, State, Deployment, Activity diagram, ...

```plantuml
@startuml
title OK on Class, State, Deployment

label l1 [{{
<style>
.green {
  BackgroundColor lightgreen
}
.pink {
  BackgroundColor pink
}
.dotted {
  LineStyle 10
}
</style>
title Composition of stereo
node Green <<green>>
node Pink <<pink>>
node Dotted  <<dotted>>
node "pink + dotted" <<pink>> <<dotted>>
}}]

label l2 [{{
<style>
.green {
  BackgroundColor lightgreen
}
.pink {
  BackgroundColor pink
}
.dotted {
  LineStyle 10
}
</style>
title Composition of stereo
state Green <<green>>
state Pink <<pink>>
state Dotted  <<dotted>>
state "pink + dotted" <<pink>> <<dotted>>
}}]

label l3 [{{
<style>
.green {
  BackgroundColor lightgreen
}
.pink {
  BackgroundColor pink
}
.dotted {
  LineStyle 10
}
</style>
title Composition of stereo
class Green <<green>>
class Pink <<pink>>
class Dotted  <<dotted>>
class "pink + dotted" <<pink>> <<dotted>>
}}]
@enduml
```
>[![](https://www.planttext.com/api/plantuml/svg/pPEnQiCm48PtFSMHDI5jsbMOK0SwfD12nj91ZQwEYAmpqcK8nkzUozU4XdXejwpVVzP_VBAMYKtaG-qLE_O8hskW0AKtASrWmuPn1YjiF1rh3AoKDv_emI_WlMsLJdpqMAXv5H43j0hWoMptLQH3i2LvYk1TjUCXLfsQDoxiHyToa0SiCQCTHbvSm4q-7nRtVQJlVifapvBgXf9ZrszBNv0O8v8AP16UXsMq7bgBGk1RRjOwzvpGIhfwAgqdFXcsd8BGoVchtuDTzt5syN2RBbA-l2iPGYzj21lH8S5_VJpUfezj_gUlV0Yzz25in8S4V_Mnn63xX_KD)](https://www.planttext.com/?text=pPEnQiCm48PtFSMHDI5jsbMOK0SwfD12nj91ZQwEYAmpqcK8nkzUozU4XdXejwpVVzP_VBAMYKtaG-qLE_O8hskW0AKtASrWmuPn1YjiF1rh3AoKDv_emI_WlMsLJdpqMAXv5H43j0hWoMptLQH3i2LvYk1TjUCXLfsQDoxiHyToa0SiCQCTHbvSm4q-7nRtVQJlVifapvBgXf9ZrszBNv0O8v8AP16UXsMq7bgBGk1RRjOwzvpGIhfwAgqdFXcsd8BGoVchtuDTzt5syN2RBbA-l2iPGYzj21lH8S5_VJpUfezj_gUlV0Yzz25in8S4V_Mnn63xX_KD)

## :x: KO on Mindmap, WBS, ~Activity diagram~, ...
```plantuml
@startuml
title KO on Mindmap, WBS, Activity diagram, ...
label l1 [{{mindmap
<style>
.green {
  BackgroundColor lightgreen
}
.pink {
  BackgroundColor pink
}
.dotted {
  LineStyle 10
}
</style>
* Colors
** Green <<green>>
** Pink <<pink>>
** Dotted  <<dotted>>
** pink + dotted \n(KO on 1.2023.11beta2)<<pink>> <<dotted>>
}}]

label l2 [{{
<style>
.green {
  BackgroundColor lightgreen
}
.pink {
  BackgroundColor pink
}
.dotted {
  LineStyle 10
}
</style>
:Colors;
:Green; <<green>>
:Pink; <<pink>>
:Dotted;  <<dotted>>
:pink + dotted
(KO on 1.2023.11beta2); <<pink>> <<dotted>>
}}]

label l3 [{{wbs
<style>
.green {
  BackgroundColor lightgreen
}
.pink {
  BackgroundColor pink
}
.dotted {
  LineStyle 10
}
</style>
* Colors
** Green <<green>>
** Pink <<pink>>
** Dotted  <<dotted>>
** pink + dotted \n(KO on 1.2023.11beta2)<<pink>> <<dotted>>
}}]
@enduml
```
>[![](https://www.planttext.com/api/plantuml/svg/vPCnRy8m48Lt_ufRAW55BWbR4Y4AbJgq5PKOEa07XrnJ2yT6pa654F-ziGq2L3ftw6RTk_UUyoaUrYGihIl5I992U9g0qV0YTL69LHVUHjCkt2z8RYHjeP2YjABg0kUSAP6Z0XN1RBUhmZxBQjeg731UMaGDEmOm4ejbQSrQ5sEZZ0Kboq_oCjiplf9wUNND2MwXC4HO-9LdgN7gyY7gDL9sToZhWBVKhDE1Hz-RPRvWC72ZLrUHPIum31v2P3CBuM7gRt8BXxwvRWKI4OzxSPz7KOuaulOnvzoytx-p8utOqVWR34cWaBB4GqZFA2IEGNg2a0G4wGM3v880k_xzfv3V2FGTWQ-y_h__meSnX1OcUOrsq_X_EETwlBOMDS46RIqR_UPmQBNP47NHFCTl)](https://www.planttext.com/?text=vPCnRy8m48Lt_ufRAW55BWbR4Y4AbJgq5PKOEa07XrnJ2yT6pa654F-ziGq2L3ftw6RTk_UUyoaUrYGihIl5I992U9g0qV0YTL69LHVUHjCkt2z8RYHjeP2YjABg0kUSAP6Z0XN1RBUhmZxBQjeg731UMaGDEmOm4ejbQSrQ5sEZZ0Kboq_oCjiplf9wUNND2MwXC4HO-9LdgN7gyY7gDL9sToZhWBVKhDE1Hz-RPRvWC72ZLrUHPIum31v2P3CBuM7gRt8BXxwvRWKI4OzxSPz7KOuaulOnvzoytx-p8utOqVWR34cWaBB4GqZFA2IEGNg2a0G4wGM3v880k_xzfv3V2FGTWQ-y_h__meSnX1OcUOrsq_X_EETwlBOMDS46RIqR_UPmQBNP47NHFCTl)

Related or pseudo similar issue:
- #1422 

_[Initial request from Discord from m42e]_
Regards,
Th.

**Comment by The-Lum** [comment]
Hello PlantUML team,

FYI:
- that is now OK for Activity diagram ✔️ 
- remains only Mindmap or WBS... ❌ 

```puml
@startuml
title KO on Mindmap, WBS, ...
label l1 [{{mindmap
<style>
.green {
  BackgroundColor lightgreen
}
.pink {
  BackgroundColor pink
}
.dotted {
  LineStyle 10
}
</style>
* Colors
** Green <<green>>
** Pink <<pink>>
** Dotted  <<dotted>>
** pink + dotted \n(KO on 1.2024.6)<<pink>> <<dotted>>
}}]

label l2 [{{wbs
<style>
.green {
  BackgroundColor lightgreen
}
.pink {
  BackgroundColor pink
}
.dotted {
  LineStyle 10
}
</style>
* Colors
** Green <<green>>
** Pink <<pink>>
** Dotted  <<dotted>>
** pink + dotted \n(KO on 1.2024.6)<<pink>> <<dotted>>
}}]
@enduml
```
>[![](https://www.planttext.com/api/plantuml/svg/vL9DIyCm6BxdLxmNOQiZhaKyXJ8smOFAX1qyE0-jVQrXwPkHf1Cf_U_ceuCT_0VUmlEPF6HbNMLStodcf5C8ZrlG1C-IcguwBk1rlLi0vvofgaO5AeUtOUWIouHrFmfBnbk3I30mW7NrSMYDxgdPQAKDADb-kKYpaV6ZfCEVia04GQETmoPAdYJXBkH3llIKk9dACeWMox8C7cAl4B6WB0Fq4Yg42849k4-H7alX2OqtkOQfRq-pzEwS5ylYbj_DpmcNjd5yP-STYh33Tst_umQVMZiqiAqjcfDtNdZsjEcDGN9mGcEbPwwcmspELaYD_sY_)](https://www.planttext.com/?text=vL9DIyCm6BxdLxmNOQiZhaKyXJ8smOFAX1qyE0-jVQrXwPkHf1Cf_U_ceuCT_0VUmlEPF6HbNMLStodcf5C8ZrlG1C-IcguwBk1rlLi0vvofgaO5AeUtOUWIouHrFmfBnbk3I30mW7NrSMYDxgdPQAKDADb-kKYpaV6ZfCEVia04GQETmoPAdYJXBkH3llIKk9dACeWMox8C7cAl4B6WB0Fq4Yg42849k4-H7alX2OqtkOQfRq-pzEwS5ylYbj_DpmcNjd5yP-STYh33Tst_umQVMZiqiAqjcfDtNdZsjEcDGN9mGcEbPwwcmspELaYD_sY_) 

Regards,
Th.

**Comment by The-Lum** [comment]
Seems to be fixed by:
- https://github.com/plantuml/plantuml/commit/35600a0c1dd90491238ff89f2175ba6e8549a5e3

Thanks to @arnaudroques

**Comment by The-Lum** [comment]
This is now fixed on `v1.2024.7`, Thanks for the works 👍 

>[![](https://www.plantuml.com/plantuml/svg/vL9DIyCm6BxdLxmNOQiZhaKyXJ8smOFAX1qyE0-jVQrXwPkHf1Cf_U_ceuCT_0VUmlEPF6HbNMLStodcf5C8ZrlG1C-IcguwBk1rlLi0vvofgaO5AeUtOUWIouHrFmfBnbk3I30mW7NrSMYDxgdPQAKDADb-kKYpaV6ZfCEVia04GQETmoPAdYJXBkH3llIKk9dACeWMox8C7cAl4B6WB0Fq4Yg42849k4-H7alX2OqtkOQfRq-pzEwS5ylYbj_DpmcNjd5yP-STYh33Tst_umQVMZiqiAqjcfDtNdZsjEcDGN9mGcEbPwwcmspELaYD_sY_)](https://www.plantuml.com/plantuml/uml/vL9DIyCm6BxdLxmNOQiZhaKyXJ8smOFAX1qyE0-jVQrXwPkHf1Cf_U_ceuCT_0VUmlEPF6HbNMLStodcf5C8ZrlG1C-IcguwBk1rlLi0vvofgaO5AeUtOUWIouHrFmfBnbk3I30mW7NrSMYDxgdPQAKDADb-kKYpaV6ZfCEVia04GQETmoPAdYJXBkH3llIKk9dACeWMox8C7cAl4B6WB0Fq4Yg42849k4-H7alX2OqtkOQfRq-pzEwS5ylYbj_DpmcNjd5yP-STYh33Tst_umQVMZiqiAqjcfDtNdZsjEcDGN9mGcEbPwwcmspELaYD_sY_)
